### DISCUSSION TITLE
What's wttr short for?

### DISCUSSION THREAD

**Discussion by playgithub** [chubin/wttr.in#1000] [Q&A] [answered]
What's wttr short for?

**Comment by darkman7hr** [comment]
weather?:)

**Reply by playgithub** [reply]
only one "t" in "weather", but two "t"s in "wttr"

**Comment by darkman7hr** [comment]
in english maybe :)

**Reply by playgithub** [reply, marked as answer]
"Das Wetter" in German

**Reply by chubin** [reply]
Yes, absolutely. Wttr is Wetter in German and, with minor differences in spelling, in all Germanic languages (incl. English):

Wetter, Weather, Weer, Wedder, etc. And wttr too :)
